portfolio/src/main/webapp/youtube.html

```diff
@@ -9,7 +9,7 @@
   <body>
       <form action="/data" method="POST">
         <label for="user-input">What's a youtube video idea you have for me?</label><br>
-        <textarea name="user-input">Put your idea here!</textarea>
+        <textarea name="user-input" required></textarea>
```

Comment:
You'll need to add an id attribute for the label to be associated with the textarea. So `<textarea id="user-input" name="user-input" ....>`